src/BloomExe/Browser.cs

```diff
@@ -441,7 +441,13 @@ private void OnConsoleMessage(object sender, ConsoleMessageEventArgs e)
 			if(e.Message.StartsWith("[JavaScript Warning"))
 				return;
 			if (e.Message.StartsWith("[JavaScript Error"))
-				ReportJavaScriptError(new GeckoJavaScriptException(e.Message));
+			{
+				// BL-4737 Don't report websocket errors, but do send them to the Debug console.
+				if (!e.Message.Contains("has terminated unexpectedly. Some data may have been transferred."))
```

Comment:
Hmm... any chance this will get localized?
(Reviewing my own code the next day...)